app/models/enterprise.rb

```diff
@@ -19,6 +19,7 @@ class Enterprise < ApplicationRecord
   preference :shopfront_producer_order, :string, default: ""
   preference :shopfront_order_cycle_order, :string, default: "orders_close_at"
   preference :shopfront_product_sorting_method, :string, default: "by_category"
+  preference :product_stock_display, :boolean, default: false
```

Comment:
Maybe `:shopfront_low_stock_display` would be more accurate? We are not displaying stock all the time.